calico/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -0,0 +1,164 @@
+---
+description: Understand how tiered policy works and supports microsegmentation.
+---
+
+# Get started with policy tiers
+
+## Seamless network policy integration
+
+**Network policy** is the primary tool for securing a Kubernetes network. It lets you restrict network traffic in your cluster so only the traffic that you want to flow is allowed. {{prodname}} provides more robust policy than Kubernetes, but you can use them together -- seamlessly. {{prodname}} supports:
+
+- {{prodname}} network policy, (namespaced)
+- {{prodname}} global network policy (non-namespaced, global)
+- Kubernetes network policy
+
+## Tiers: what and why?
+
+**Tiers** are a hierarchical construct used to group policies and enforce higher precedence policies that cannot be circumvented by other teams. As you will learn in this tutorial, tiers have built-in features that support workload microsegmentation.
+
+All {{prodname}} and Kubernetes network policies reside in tiers. You can start "thinking in tiers" by grouping your teams and types of policies within each group. For example, we recommend these three tiers (platform, security, and application).
+
+![policy-types](/img/calico-enterprise/policy-types.png)
+
+Next, you can determine the priority of policies in tiers (from top to bottom). In the following example, the platform and security tiers use {{prodname}} global network policies that apply to all pods, while developer teams can safely manage pods within namespaces using Kubernetes network policy for their applications and microservices.
+
+![policy-tiers](/img/calico-enterprise/policy-tiers.png)
+
+## Create a tier
+
+The following is a sample YAML that creates a security tier and uses `kubectl` to apply it.
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: Tier
+metadata:
+  name: security
+spec:
+  order: 300
+```
+
+```bash
+kubectl apply -f security.yaml
+```
+
+## The default tier:
+
+The default tier is created during installation and has the order of 1,000,000.
+
+The default tier is where:
+
+- You manage all Kubernetes network policies
+- All network and global network policies without an explicit tier are placed.
+- Network and global network policies are placed when you upgrade from Project Calico without tier support to {{prodname}} release with tier support.
+
+## AdminNetworkPolicy tier:
+
+The AdminNetworkPolicy tier is created during installation and has the order of 1,000.
+
+The AdminNetworkPolicy tier is where:
+
+- You manage all Kubernetes admin network policies
+
+## Tier order
+
+Tiers are ordered from left to right, starting with the highest priority (also called highest precedence) tiers.
```

Comment:
left to right is a UI concept. We do not have anything like in OSS and we cannot reference a UI image